website/docs/events/m4-slides/.slides/methodology.md

```diff
@@ -1,25 +1,81 @@
 ---
-
 ## Methodology
 
-Owner: RăzvanN
+----
+
+### Methodology
+
+- content creation
+- infrastructure used for deployment, testing and publishing
+
+----
+
+### Content Creation
+
+- types of content
+- how is the content used
+- guidelines on content creation
+- guidelines on content organization
+- guidelines on contribution procedures
+- technologies used
 
-probably follow structure in [the `methodology` repository](https://github.com/open-education-hub/methodology/), apart from infrastructure
+----
+
+### Types of Content
+
+- reading
+- media
+- guides
+- drills
+- slides
+- projects
+
+----
+
+### Using Content
+
+|Activity           | Content Types used                     |
+|-------------------|----------------------------------------|
+|lectures           | slides, media, guides, drills          |
+|practical sessions | slides, reading, media, guides, drills |
+|assignments        | projects, drills                       |
+|self-study         | reading, media, drills, projects       |
+|tests              | drills                                 |
 
 ----
 
-### TODO
+### Repository Structure
 
-TODO
+```bash
+|-- chapter1
+|   |-- topic1
+|   |-- topic2
+|   |   |-- reading
+|   |   |-- media
+|   |   |-- slides
+|   |   |-- guides
+|   |   |-- drills
+|   |   `-- projects
+|-- chapter2
+|-- chapter3
+   ...
+`-- chapterN
+```
 
 ----
 
-### TODO
+### Technologies used
 
-TODO
+- Github for storing content
+- Markdown for everything
+- Draw.io for images and diagrams
+- reveal-md for slide rendering
+- others
 
 ----
 
```

Comment:
Add a section called "Challenges and Road Ahead" or something along that way. Mention things we are unsure of, and items we want / need to tackle in the future for the methodology.